Makefile

```diff
@@ -16,7 +16,7 @@ docs:
 .PHONY: docs-commands-data
 docs-commands-data: docs
 	git -C docs pull
-	git -C docs checkout master
+	git -C docs checkout main
```

Comment:
This is for the documentation repo, AFAIK they still are on the `master` branch.